packages/components-props-analyzer/package.json

```
@@ -20,7 +20,7 @@
         "compile": "node ../../scripts/build",
         "gen:props": "ts-node --project ./bin/tsconfig.json ./bin/index.ts",
         "lintfix": "../../node_modules/.bin/prettier --write \"**/*.{scss,ts,tsx,js,jsx,json,md,yml,html}\" && ../../node_modules/.bin/eslint -c ./.eslintrc.js \"src/**/*.{ts,tsx}\" --fix",
-        "prettify": "prettier --write 'src/components/*.ts'",
+        "prettify": "prettier --write \"src/components/*.ts\"",
```

For the tsx extension.

```suggestion
        "prettify": "prettier --write \"src/components/*.ts*\"",
```

---

Good idea

---

Actually, it looks like it's only `ts` files in that folder...